src/Hazelcast.Net/Events/ClientLifecycleState.cs

```diff
@@ -16,6 +16,13 @@ namespace Hazelcast.Events
 {
     public enum ClientLifecycleState
     {
+        // Active    Connecting
+        // Active    Connected
+        // NotActive Disconnecting
+        // Active    Reconnecting (?)
+        // NotActive Disconnected
+        // 
+
         Starting,
```

Comment:
Are we firing all of these State events or is it in another PR?

Reply:
Another PR.